releases/release-1.30/release-notes/maps/pr-122013-map.yaml

```diff
@@ -0,0 +1,9 @@
+pr: 122013
+releasenote:
+  text: "A deprecated flag `--pod-max-in-unschedulable-pods-duration` was initially
+    planned to be removed in v1.26, but we have to change this plan. We found [an
+    issue](https://github.com/kubernetes/kubernetes/issues/110175) in which Pods can
+    be stuck in the unschedulable pod pool for 5 min, and using this flag is the only
+    workaround for this issue. \nThis issue only could happen if you use custom plugins
```

Comment:
Should we add a new line here instead of `\n`? Or will it be correctly formatted when rendered?